versions/1.0.0.md

```diff
@@ -92,12 +92,12 @@ This object represents one or more changes to be applied to the target document
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="action-target"></a>target | `string` | **REQUIRED** A JSONPath query expression referencing the target objects in the target document. |
+| <a name="action-target"></a>target | `string` | **REQUIRED** A JSONPath expression selecting nodes in the target document. |
 | <a name="action-description"></a>description | `string` | A description of the action. [[CommonMark]] syntax MAY be used for rich text representation. |
-| <a name="action-update"></a>update | Any | An object with the properties and values to be merged with the object(s) referenced by the `target`. This property has no impact if `remove` property is `true`. |
+| <a name="action-update"></a>update | Any | If the `target` selects an object node, the value of this field should be an object with the properties and values to merge with the node. If the `target` selects an array, the value of this field should be an entry to append to the array. This field has no impact if the `remove` field of this action object is `true`. |
 | <a name="action-remove"></a>remove | `boolean` | A boolean value that indicates that the target object is to be removed from the the map or array it is contained in. The default value is `false`. |
 
-The result of the `target` JSONPath query expression must be zero or more objects or arrays (not primitive types or `null` values). If you wish to update a primitive value such as a string, the `target` expression should select the _containing_ object in the target document.
+The result of the `target` JSONPath expression must be zero or more objects or arrays (not primitive types or `null` values). If you wish to update or remove a primitive value such as a string, the `target` expression should select the _containing_ object in the target document.
```

Comment:
```suggestion
The result of the `target` JSONPath expression must be zero or more objects or arrays (not primitive types or `null` values).

To update a primitive property value such as a string, the `target` expression should select the _containing_ object in the target document and `update` should contain an object with the property and its new primitive value.

Similarly for arrays of primitive values: to replace or remove a primitive array item, the `target` expression should select the  object containing the array-valued property and `update` should contain an object with the property and its new array value.
```

Reply:
I think the operation you describe in the last paragraph is simply a replace of an array. Characterizing this as replacing or removing a primitive array item seems misleading to me.

Reply:
Ok, how about the modified proposal?

Reply:
That looks good! Thank you. Pls commit that and then I'll give it one last pass.